src/components/refactor/dialogs/voltage-level-creation/bus-bar-section/bus-bar-section-creation.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { gridItem } from 'components/dialogs/dialogUtils';
+import IntegerInput from 'components/refactor/rhf-inputs/integer-input';
+import TextInput from 'components/refactor/rhf-inputs/text-input';
+import {
+    BUS_BAR_SECTIONS,
+    HORIZONTAL_POSITION,
+    ID,
+    NAME,
+    VERTICAL_POSITION,
+} from 'components/refactor/utils/field-constants';
+
+export const numericalWithButton = {
+    type: 'number',
+    inputProps: { min: 0, style: { textAlign: 'right' } },
```

Comment:
since we are using this, we should remove "isInputPositiveOnly" from integerInput to avoid confusion for future developments